components/Trade/Advanced/TradingPanel/MarketSelect/index.tsx

```diff
@@ -116,36 +125,19 @@ const MarketSelectDropdownButton: React.FC<MarketSelectDropdownButtonProps> = st
     @media (max-width: 1600px) {
         height: 22px;
     }
-
-    &:hover {
-        cursor: pointer;
-    }
-
-    > span {
-        margin-left: auto;
-    }
-
-    > .down-arrow {
-        margin: auto 0 auto auto;
-        width: 1em;
-        height: 1em;
-        display: inline-block;
-        transition: 0.3s;
-        transform: ${(props) => (props.arrowUp ? 'rotate(180deg) translateY(-2px)' : 'translateY(-1px)')};
-    }
 `;
 
 const MarketContainer = styled.div`
     font-size: var(--font-size-medium);
-    letter-spacing: -0.4px;
+    letter-spacing: var(--letter-spacing-extra-small);
     display: flex;
     height: var(--height-small-container);
 `;
 
 const SBox = styled(Box)<{
     $display: boolean;
 }>`
-    background-color: ${(props) => props.color as string}!important;
+    background-color: ${(props: any) => props.color as string}!important;
```

Comment:
if there is something in props add it. You will have to check if my suggestion actually works